scripts/run_tests.sh

```diff
@@ -36,6 +36,7 @@ for dir in $(find $TARGET -type d -name "testdata"); do
     test_snapshot=${test_snapshot%.*}.yml
     filename=$(basename $file)
 
+    # docker rmi bearer/bearer:latest-amd64
```

Comment:
Is this intentionally included?

Reply:
Yes that is intentional (note to remove the previous version before running locally) ;)